src/Parser/Monk/Mistweaver/Modules/Features/AlwaysBeCasting.js

```diff
@@ -78,27 +78,24 @@ class AlwaysBeCasting extends CoreAlwaysBeCastingHealing {
     return {
       actual: this.downtimePercentage,
       isGreaterThan: {
-        minor: 0.2,
-        average: 0.35,
+        minor: 0.4,
+        average: 0.55,
         major: 1,
       },
       style: 'percentage',
     };
   }
 
   suggestions(when) {
-    const nonHealingTimePercentage = this.totalHealingTimeWasted / this.owner.fightDuration;
-    const deadTimePercentage = this.totalTimeWasted / this.owner.fightDuration;
-
-    when(nonHealingTimePercentage).isGreaterThan(0.5)
+    when(this.nonHealingTimePercentage).isGreaterThan(0.4)
```

Comment:
Please consider using the threshold properties instead of the threshold numbers for the isGreaterThan, regular and major methods to avoid them getting out of sync.

Reply:
Yea, I misunderstood the other comment you made on the original PR. 

I have updated the suggestions through to leverage the `suggestionThresholds` getter.